test-base.cfg

```diff
@@ -208,19 +208,26 @@ commands =
 
 # bin/pycodestyle : pycodestyle validation of the packages source.
 [pycodestyle]
-recipe = zc.recipe.egg
+recipe = zc.recipe.egg:scripts
 eggs = pycodestyle
-args = ['--exclude=tests', '--config=${pycodestyle-cfg:cfgfile}']
```

Comment:
But if I'm reading it correctly, it is not included in the excludes 😉 :
`exclude = bootstrap.py,pyxbgen.py,bindings,skins,monkey`

So this PR also starts validating files in `tests`, right?
That's ok, but I'd just have expected the PR body to have a note on this fact.